v2.go

```diff
@@ -159,7 +168,13 @@ func (header *Header) formatVersion2() ([]byte, error) {
 	buf.WriteByte(header.TransportProtocol.toByte())
 	// When UNSPEC, the receiver must ignore addresses and ports.
 	// Therefore there's no point in writing it.
```

Comment:
...but if TLVs are present, those should be written. Please, augment this comment.

Reply:
Ahh, sorry I forgot to change this comment